mewarpx/mewarpx/util.py

```diff
@@ -4,9 +4,19 @@
 import errno
 import inspect
 import os
+from pywarpx.picmi import constants
 
 from pywarpx import geometry
 
+# CONSTANTS - SI
```

Comment:
Let's put these in a new file `mwxconstants.py` unless you object... I came to regret having them all in `util` in `minerva`.

Reply:
I guess `util` will have to import `mwxconstants` at that point... that's probably OK though, `mwxconstants` should never need anything other than `pywarpx.picmi` and third-party packages so we don't get any import loops.